specs/waku/waku-v2.md

```diff
@@ -257,24 +252,18 @@ NOTE I would imagine this is implied from the contentFilter, especially as two n
 
 
 ```protobuf
-message RPC {
```

Comment:
This RPC could be at the top level, and used for more than one spec, as I commented

Reply:
And then what would its contents by `bytes` or a `oneof` field?